test/integration/image-component/default/pages/lazy-src-change.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import Image from 'next/image'
+
+const Page = () => {
+  const [src, setSrc] = React.useState('/test.jpg')
```

Comment:
Does the bug reproduce with `pages/lazy-src-change.js`?

I thought it would need to be `pages/ lazy-src-change/[id].js` so you can do a page navigation but reusing the same Image instance?

Reply:
> Does the bug reproduce with `pages/lazy-src-change.js`?
> 
> I thought it would need to be `pages/ lazy-src-change/[id].js` so you can do a page navigation but reusing the same Image instance?

Yes, changing src does reproduce the issue. The `page/[id]` is derived where I encounter the issue.